pkgs/top-level/all-packages.nix

```diff
@@ -16541,11 +16541,16 @@ with pkgs;
     };
 
   mpv = callPackage ../applications/video/mpv rec {
-    lua = lua5_1;
-    lua5_sockets = lua5_1_sockets;
+    inherit (luaPackages) luasocket;
     youtube-dl = pythonPackages.youtube-dl;
-    libva = libva-full;
-    waylandSupport = stdenv.isLinux;
+    libva      = libva-full;
+    waylandSupport     = stdenv.isLinux;
+    alsaSupport        = !stdenv.isDarwin;
+    pulseSupport       = !stdenv.isDarwin;
+    rubberbandSupport  = !stdenv.isDarwin;
+    dvdreadSupport     = !stdenv.isDarwin;
+    dvdnavSupport      = !stdenv.isDarwin;
+    drmSupport         = !stdenv.isDarwin;
```

Comment:
drmSupport is already disabled.

Reply:
Oh, right. So does this mean the `? true` in the package is overriding the option in the call?

Reply:
Only if the variable is null, which is not the case.
Does `why-depends` only show the first match?

```console
$ nix-repl> ({ a ? true }: a) { a = false; }
false
```

Reply:
```console
$ nix why-depends -a nixpkgs.mpv nixpkgs.xlibs.libpciaccess
```

gives a more complete picture